src/main/java/io/github/cottonmc/cotton/gui/widget/WWidget.java

```diff
@@ -362,4 +362,11 @@ public WWidget hit(int x, int y) {
 	 */
 	@Environment(EnvType.CLIENT)
 	public void tick() {}
+	
+	/**
+	 * Should this WWidget be included when the layout renderer generates a layout ?
+	 */
+	public boolean shouldExpandToFit() {
```

Comment:
`shouldParentExpandToFit` would be a better name since it's not this widget expanding.